packages/action-tracker/src/index.ts

```diff
@@ -32,13 +47,23 @@ export const buildEmbraceMiddleware = () => {
           );
         } else {
           const endTime = new Date().getTime();
-          NativeModules.EmbraceManager.logRNAction(
-            action.type.toString().toUpperCase(),
+          attributePromises.push(
+            NativeModules.EmbraceManager.addSpanAttributeToSpan(
+              spanId,
+              "outcome",
+              "SUCCESS",
+            ),
+          );
+
+          await Promise.all(attributePromises);
+          NativeModules.EmbraceManager.recordCompletedSpan(
```

Comment:
should be `stopSpan` here, `recordCompletedSpan` wouldn't close the one that was started above

Reply:
Yes this was the example that I gave you through slack, I reverted it to let stopSpan